VRDR/ValueSets.cs

```diff
@@ -3,86 +3,51 @@ namespace VRDR
     /// <summary> ValueSet Helpers </summary>
     public static class ValueSets
```

Comment:
This whole file is autogenerated with the scripts right? If so there should be a warning that says not to edit this file